docs/rules/max-attributes-per-line.md

```diff
@@ -0,0 +1,88 @@
+# Define the number of attributes allows per line (max-attributes-per-line)
+
+Limits the maximum number of props per line to improve readability
+
+
+## Rule Details
+
+This rule aims to enforce a number of props per line in templates. It checks all the elements in a template and verifies that the number of props per line does not exceed the defined maximum.
```

Comment:
Again `attributes`, unless we only care about components' props. But I think we should care about every attribute. Prop is just another type of property/attribute.